test_scripts/Polices/appID_Management/163_ATF_P_TC_HMI_Status_Value_Of_AppId_In_PT_Is_Null.lua

```diff
@@ -75,7 +75,8 @@ end
 
 function Test:Precondition_UpdatePolicy()
   testCasesForPolicyAppIdManagament:updatePolicyTable(self, "files/jsons/Policies/appID_Management/ptu_013_2.json")
-  self.mobileSession2:ExpectNotification("OnHMIStatus", { hmiLevel ="NONE", systemContext = "MAIN", audioStreamingState = "NOT_AUDIBLE" })
+  EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", { appID =  data.params.application.appID, appRevoked =  true})
```

Comment:
@pold500, Value of data.params.application.appID will be nil, since data is not defined.
Please use value from self.applications[] table depending on which application is revoked